openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardNew/CommentCard.component.tsx

```diff
@@ -0,0 +1,150 @@
+}
+
+const CommentCard = ({ feed, post, isLastReply }: CommentCardInterface) => {
+  const { updateFeed } = useActivityFeedProvider();
+  const [isHovered, setIsHovered] = useState(false);
+  const [isEditPost, setIsEditPost] = useState<boolean>(false);
+  const [postMessage, setPostMessage] = useState<string>('');
+  const seperator = '.';
+
+  const onEditPost = () => {
+    setIsEditPost(!isEditPost);
+  };
+
+  const onUpdate = (message: string) => {
+    const updatedPost = { ...feed, message };
+    const patch = compare(feed, updatedPost);
+    updateFeed(feed.id, post.id, false, patch);
+    setIsEditPost(!isEditPost);
+  };
+
+  const handleSave = useCallback(() => {
+    onUpdate?.(postMessage ?? '');
+  }, [onUpdate, postMessage]);
+
+  const getDefaultValue = (defaultMessage: string) => {
+    return MarkdownToHTMLConverter.makeHtml(getFrontEndFormat(defaultMessage));
+  };
+
+  const feedBodyRender = useMemo(() => {
+    if (isEditPost) {
+      return (
+        <ActivityFeedEditor
+          focused
+          className="mb-8 reply-feed-editor"
+          defaultValue={getDefaultValue(post.message)}
+          editorClass="is_edit_post"
+          onSave={handleSave}
+          onTextChange={(message) => setPostMessage(message)}
+        />
+      );
+    }
+
+    return null;
+  }, [isEditPost, postMessage, handleSave]);
+
+  function stripHtml(html: any) {
+    const tempDiv = document.createElement('div');
+    tempDiv.innerHTML = html;
+
+    return tempDiv.innerText || tempDiv.textContent;
+  }
+
+  return (
+    <div
+      className={classNames('d-flex justify-start relative reply-card', {
+        'reply-card-border-bottom': !isLastReply,
+      })}
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}>
+      <div className="profile-picture m-r-xs">
+        <ProfilePicture
+          avatarType="outlined"
+          key={feed.id}
+          name={feed.updatedBy!}
+          size={32}
+        />
+      </div>
+      <div>
+        <div className="d-flex items-center gap-2 flex-wrap">
+          <Typography.Text
+            className={` 
+                              activity-feed-user-name reply-card-user-name'
+                            `}>
+            {feed.updatedBy}
+          </Typography.Text>
+          <Typography.Text className="seperator">{seperator}</Typography.Text>
+          <Typography.Text>
+            <Tooltip
+              color="white"
+              overlayClassName="timestamp-tooltip"
+              title={formatDateTime(post.postTs)}>
+              <span
+                className="feed-card-header-v2-timestamp mr-2"
+                data-testid="timestamp">
+                {getRelativeTime(post.postTs)}
+              </span>
+            </Tooltip>
+          </Typography.Text>
+        </div>
+        {isEditPost ? (
+          feedBodyRender
+        ) : (
+          <Text className="reply-message">{stripHtml(post.message)}</Text>
+        )}
```

Comment:
Can't we pull this within feedBodyRender?

Reply:
done